src/main.c

```diff
@@ -315,6 +332,21 @@ char simulator_tone[5];
 char qsos[MAX_QSOS][LOGLINELEN+1];
 int nr_qsos = 0;
 
+int qsoflags_for_qtc[MAX_QSOS];
+int nr_qsosflags_for_qtc;
+int next_qtc_qso;
+t_qtclist qtclist;
+int nr_qtcsent = 0;
+t_qtcreclist qtcreclist;
+struct t_qtc_store_obj *qtc_temp_obj;
+int qtcdirection = 0;
+//char qtc_ry_lines[12][50] = {"", "", "", "", "", "", "", "", "", "", "", ""};
+//int qtc_ry_lines_attr[12] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+t_qtc_ry_line qtc_ry_lines[QTC_RY_LINE_NR];
+int qtc_ry_currline = 0;
+int qtc_ry_capture;
+int qtc_ry_copied;
+
```

Comment:
As global variables makes the code much harder to maintain, please check if the new qtc related definitions (line 335..348) are really need in a global context. Maybe they can be moved to the according *.[ch] files or make local variables.

An example for the last one seems to be 'qtc_temp_store'.